namer/namer.cc

```diff
@@ -1322,11 +1430,21 @@ class SymbolDefiner {
                 // We need alias methods in the FoundDefinitions list not so that we can actually
                 // create method symbols for them yet, but just so we can know which alias methods
                 // to delete on the fast path. Alias methods will be defined later, in resolver.
+                //
+                // We still need to put something here so that other found definitions that
+                // reference methods will get the correct symbols.
+                definedMethods.emplace_back(core::MethodRef{});
```

Comment:
What do you mean by this? ooc, was there a test that failed unless you did this?

Reply:
When I wrote the comment, I meant that you could run into a situation where field X had method owner M at INDEX...but if there were some alias methods defined prior to M, then we wouldn't actually have table entries for those alias methods, and then INDEX would point at the wrong method when we went to go define field X.  And no `ENFORCE`s would catch this because the indices were still in bounds--at least for the testcases where things were going wrong; I guess you could write a testcase that tries to catch that?

rbi generation failed because we weren't putting refs in for alias methods.